src/demo/helpers.js

```diff
@@ -105,3 +106,17 @@ export const filterFishComponents = (fishComponents, dataSet) => {
 
   return filteredCopy;
 };
+
+export const getAppMode = () => {
+  const appMode = _.last(queryStrFor('mode').toLowerCase().split('-'));
```

Comment:
i pulled your branch to run locally, and it looks like we just need a safeguard here:

if we load a page without `?mode=` set, this line throws an error:
![Screen Shot 2019-11-01 at 2 42 16 PM](https://user-images.githubusercontent.com/9812299/68058349-06cf3100-fcb6-11e9-9b1b-06039bb01f1c.png)
![Screen Shot 2019-11-01 at 2 41 43 PM](https://user-images.githubusercontent.com/9812299/68058350-0767c780-fcb6-11e9-9cc1-a39d430deacf.png)